doc/release-notes-19464.md

```diff
@@ -0,0 +1,7 @@
+Updated settings
+----------------
+
+- The `-banscore` configuration option, which modified the default threshold for
+  disconnecting and discouraging misbehaving peers, has been removed as part of
+  changes in this release to the handling of misbehaving peers. Refer to the
+  section, "Changes regarding misbehaving peers", for details. (#19464)
```

Comment:
Mind adding this to the main release notes file? There shouldn't be any conflicts and this is not for backport either.

Reply:
Done